interface/patient_file/history/history_sdoh_save.php

```diff
@@ -165,8 +165,8 @@
     // Return to demographics (or wherever you prefer)
     // Redirect to health concerns selection page
     $redirectUrl = $GLOBALS['webroot'] . "/interface/patient_file/history/history_sdoh_health_concerns.php"
-        . "?pid=" . urlencode($pid)
-        . "&sdoh_id=" . urlencode($id);
+        . "?pid=" . urlencode((string) $pid)
+        . "&sdoh_id=" . urlencode((string) $id);
```

Comment:
`http_build_query` is probably how we should build url parameters going forward.